src/commands/Changelog.ts

```diff
@@ -0,0 +1,318 @@
+     * @param expression The expression.
+     * @param languageAbbr Abbreviation of the language.
+     */
+    removedNodeFromLanguage(expression: ExpressionMetricMapping, languageAbbr: string) {
+        if (!this.changelog.has(expression.expression)) {
+            this.changelog.set(
+                expression.expression,
+                new ChangelogEntry(
+                    expression.expression,
+                    false,
+                    expression.languages,
+                    [],
+                    [],
+                    expression.languages
+                )
+            );
+        }
+        this.changelog.get(expression.expression)?.removedFromLanguage(languageAbbr);
+    }
+
+    /**
+     * Writes the changelog.
+     * @param metricMappings The current mappings to inform the user about which metric(s)
+     * are currently calculated with the changed syntax node(s).
+     * @return Promise that fulfills once the changelog has been written successfully.
+     */
+    writeChangelog(metricMappings: Map<string, ExpressionMetricMapping>) {
+        return new Promise<void>((resolve, reject) => {
+            const writeStream = fs.createWriteStream(pathToWriteChangelog);
+
+            writeStream.on("error", (err) => {
+                console.error("Error while writing the changelog:");
+                console.error(err);
+                reject();
+            });
+            writeStream.on("finish", () => {
+                console.log("####################################");
+                console.log('Saved overview of all changes to "' + pathToWriteChangelog + '".');
+                resolve();
+            });
+
+            writeStream.write("Changelog for updating the expression mappings" + EOL + EOL);
+            writeStream.write("New syntax nodes:" + EOL + EOL);
+            writeStream.write("Name:" + csvSeparator + "Added to language(s):" + EOL);
+
+            for (const entry of this.getAll().values()) {
+                if (entry.isNew) {
+                    writeStream.write(
+                        escapeForCsv(entry.expression) +
+                            csvSeparator +
+                            Array.from(entry.addedLanguages) +
+                            EOL
+                    );
+                }
+            }
+
+            writeStream.write(EOL + "Removed syntax nodes:" + EOL + EOL);
+            writeStream.write(
+                "Name:" +
+                    csvSeparator +
+                    "Removed from language(s):" +
+                    csvSeparator +
+                    "Used for calculating the metric(s):" +
+                    csvSeparator +
+                    "Was explicitly only activated for language(s):" +
+                    EOL
+            );
+
+            for (const entry of this.getAll().values()) {
+                if (entry.remainingLanguages.size + entry.addedLanguages.size === 0) {
```

Comment:
This could be a member function of ChangelogEntry. Pseudo Code:
```suggestion
                // Changelog class:
                // ....
                isExpressionRemoved() { return this.remainingLanguages.size === 0 && this.addedLanguages.size === 0; }
```